langchain/chains/llm.py

```diff
@@ -4,12 +4,12 @@
 from pydantic import BaseModel, Extra
 
 import langchain
-from langchain.chains.base import Chain
+from langchain.chains.base import MultiVariableChain
 from langchain.llms.base import LLM
 from langchain.prompts.base import BasePromptTemplate
 
 
-class LLMChain(Chain, BaseModel):
+class LLMChain(MultiVariableChain, BaseModel):
```

Comment:
hm so does this mean that for an llm_chain with one variable only you can no longer do `chain.run(arg)`? i dont love that....

Reply:
Isn't the user aware of the input variables for LLMChains bc you control the prompt? I see most use cases being something like:

```prompt = PromptTemplate(
    input_variables=["product"],
    template="Where can I buy {product}?",
)

from langchain.chains import LLMChain
chain = LLMChain(llm=llm, prompt=prompt)

chain.run(product="shoes") # before this would be chain.run("shoes"), doesn't seem like a huge deal to me
```